controllers/datadogagent/feature/cspm/feature_test.go

```diff
@@ -255,11 +250,11 @@ func Test_cspmFeature_Configure(t *testing.T) {
 			DDAv1:         ddav1CSPMEnabled,
 			WantConfigure: true,
 			ClusterAgent: &test.ComponentTest{
-				CreateFunc: createEmptyFakeManager,
+				CreateFunc: test.CreateEmptyFakeManager,
```

Comment:
to go a little further.
could you create a function that return a default `test.ComponentTest` 

something like

```golang
func NewDefaultComponentTest() *test.ComponentTest {
 // ...
}

func (ct *ComponentTest) WithCreateFunc(f WantFunc)  *test.ComponentTest {
  // ....
}

func (ct *ComponentTest) WithWantFunc(f WantFunc)  *test.ComponentTest {
  // ....
}

Reply:
Done :+1: